languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/added.txt

```diff
@@ -8752,3 +8753,106 @@ aliançazinha	aliançazinha	NCFS000
 aliançazinhas	aliançazinha	NCFP000
 descompressor	descompressor	NCMS000
 descompressores	descompressor	NCMP000
+home	home	NCCS000
+homes	home	NCCP000
+PIN	PIN	NCMS000
+PINs	PIN	NCMP000
+switch	switch	NCMS000
+Switch	switch	NCMS000
+SWITCH	switch	NCMS000
```

Comment:
This triple version of words (lower case, first upper case, all upper case) shouldn't be necessary. It will probably create problems in the synthesis of suggestions. 
Is anything not working as expected if these three versions are missing? The speller? Then add the three words to spelling.txt, but not to added.txt.

Reply:
Right, they're added to the hunspell files but IMO they shouldn't even be required there, which is already something I brought up with @susanaboatto when she was describing to me what was to be done here. I'll describe it in an issue. If this variation causes issues in `added.txt`, then removed they shall be.

Reply:
Thanks for noticing this @jaumeortola!

Reply:
This would eliminate the need to add capitalized and all upper case words to speller.txt. https://github.com/languagetooler-gmbh/languagetool-premium/issues/4703